src/legacy/js/functions/_postComplete.js

```diff
@@ -39,30 +39,32 @@ function completeContent(collectionId, path, recursive, redirectToPath) {
 
     var url = url + '&recursive=' + recursive;
 
+    viewCollections(collectionId);
+    
     // Update content
-    isUpdatingModal.add();
-    $.ajax({
-        url: url,
-        dataType: 'json',
-        contentType: 'application/json',
-        type: 'POST',
-        success: function () {
-            isUpdatingModal.remove();
-            if (redirect) {
-                createWorkspace(redirect, collectionId, 'edit');
-                return;
-            } else {
-                // Remove selection from 'working on: collection' tab
-                $('.js-nav-item--collection').hide();
-                $('.js-nav-item').removeClass('selected');
-                $('.js-nav-item--collections').addClass('selected');
+    // isUpdatingModal.add();
```

Comment:
It is better to delete code than to comment it out unless there's an important reason to keep it here. Otherwise it  gets messy. If it is needed then please add a comment to explain why it should be kept, thanks.

Reply:
Hi Ella, I did leave a comment in the conversation tab just underneath the commit explaining this.

"Also worth mentioning that the code changes for postComplete.js can be ignored as that was just for local testing purposes to save some time and go straight to the redirect behaviour."

I kept it in just in case anyone wanted to pull this PR to test the behaviour locally themselves to save them the trouble of constantly having to make changes to a collection prior to seeing this behaviour. 

Any code changes in _postComplete.js will be undone for the real PR 

:)